src/main/java/impl/weather/Info.java

```diff
@@ -0,0 +1,49 @@
+package impl.weather;
+
+import api.weather.DayTemperatureInfo;
+
+import java.time.Month;
+import java.util.HashMap;
+import java.util.Map;
+
+
+public class Info {
+    private final Map<Integer, DayTemperatureInfo> info = new HashMap<>();
+    private Pair averageInfo;
```

Comment:
Please, don't use Pair. Create a nested class which represents a data which you store here